File: android/app/src/main/java/com/storybooknative/stubs/ReactConfig.kt
package com.storybooknative.stubs

import com.facebook.react.bridge.BaseJavaModule
import java.util.*

class ReactConfig : BaseJavaModule() {

    override fun getName(): String {
        return "ReactConfig"
    }

    override fun hasConstants(): Boolean {
        return true
    }

    override fun getConstants(): Map<String, Any>? {
        return HashMap<String, Any>().apply {
            put("deviceId", "")
            put("operatingSystemVersion", "")
            put("cookieEid", "")
            put("isLoggedIn", "")
            put("graphqlEndPoint", "https://api.thetimes.co.uk/graphql")
            put("adNetworkId", "")
            put("timezone", "")
        }
    }
}

File: ios/storybooknative/ReactConfig.m
#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>

@interface ReactConfig : NSObject <RCTBridgeModule>
@end

@implementation ReactConfig

RCT_EXPORT_MODULE();

- (NSDictionary *)constantsToExport
{
  return @{ @"operatingSystemVersion": @"",
            @"deviceId":@"",
            @"cookieEid":@"",
            @"isLoggedIn":@"",
            @"graphqlEndPoint":@"https://api.thetimes.co.uk/graphql",
            @"adNetworkId":@"25436805",
            @"timezone":@""
            };
}

@end

